src/components/common/Actions.vue

```diff
@@ -240,6 +227,19 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
         }
       }
 
+      if (
+        !this.getEffectiveDateTime.effectiveDate ||
+        !this.isValidDateTime(this.getEffectiveDateTime.effectiveDate)
+      ) {
+        this.setEffectiveDateTimeValid(false)
+        this.setIsFilingPaying(false)
+
+        // don't call window.scrollTo during Jest tests because jsdom doesn't implement it
+        if (!this.isJestRunning) window.scrollTo({ top: 1250, behavior: 'smooth' })
+        this.setIsFilingPaying(false)
+        return
+      }
+
```

Comment:
Ignore above ^^

I don't understand how this change fixed the problem. Please explain.

Also, did you test with the other filings in this UI, and how did that work out?

Reply:
Empty date was not holding the form to be submitted. For Vol dissolution in SP and GP that is the only required field on the page. 
I will try refactor this and put this in dissolution specific files rather than being in common component.